threesixty/templates/threesixty/answer_form.html

```diff
@@ -8,11 +8,12 @@ <h2 class="w3-center">{{ name }}</h2>
     </div>
     <form method="POST" style="display: none">
         {% csrf_token %}
-        {{ form.as_p }}
+        {{ form }}
```

Comment:
is this change related? Furthermore `p` style forms are better supported in W3CSS, `__str__` defaults to `as_table`.
I would ask you to please review this change.